pages/common/az-pipelines.md

```diff
@@ -0,0 +1,33 @@
+# az pipelines
+
+> Manage Azure Pipelines resources.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/pipelines>.
+
+- Create a new Azure Pipeline (YAML based):
+
+`az pipelines create --org {{organization_url}} --project {{project_name}} --name {{pipeline_name}} --description {{description}} --repository {{repository_name}} --branch {{branch_name}}`
+
+- Delete a pipeline:
+
+`az pipelines delete --org {{organization_url}} --project {{project_name}} --id {{pipeline_id}}`
+
+- List pipelines:
+
+`az pipelines list --org {{organization_url}} --project {{project_name}}`
+
+- Queue (run) a pipeline:
```

Comment:
Perhaps this better
```suggestion
- Enqueue a specific pipeline to run:
```
Do this take too much to run? Why not just `Run` instead of `Enqueue`?

Reply:
Because pipeline is not started immediately, but assigned to an agent pool and then executed by first available agent